packages/react-data-grid/src/Canvas.js

```diff
@@ -67,6 +66,38 @@ const Canvas = createReactClass({
     isScrolling: PropTypes.bool
   },
 
+  appendScrollShim() {
```

Comment:
`ScrollShim` mixin is only used by the `Canvas` component so moved the logic into the component

Reply:
👍